test/test_HPE3ParClient_volume.py

```diff
@@ -2428,33 +2417,6 @@ def test_42_create_volume_junk_compression(self):
             optional)
         self.printFooter('create_volume_junkParams')
 
-    def test_43_create_volume_parameter_absent(self):
```

Comment:
Why is this test removed?

Reply:
@sheharai4 Found it is repeated for the case with no option and None, so removed
Added again ..